dandiapi/zarr/tasks/__init__.py

```diff
@@ -62,6 +64,11 @@ def ingest_zarr_archive(zarr_id: str, *, force: bool = False):
         # Add asset paths after ingest is finished
         add_zarr_paths(zarr)
 
+        # Set version status back to PENDING, and update modified.
+        Version.objects.filter(id=zarr.dandiset.draft_version.id).update(
+            status=Version.Status.PENDING, modified=timezone.now()
+        )
```

Comment:
To avoid an additional query -
```suggestion
        # Set version status back to PENDING, and update modified.
        Version.objects.filter(id=zarr.dandiset.draft_version_id).update(
            status=Version.Status.PENDING, modified=timezone.now()
        )
```

GitHub won't let me comment on code that hasn't been modified, but we should also have a `.select_related('dandiset')` on the query that gets this `zarr` object.

Reply:
Since `.draft_version` is just a property that calls `self.versions.filter(version='draft').get()`, `draft_version_id` can't be used here. The `select_related` call will cut down on a query, but I think otherwise we're stuck with an extra query to grab the draft version.